cypress/integration/non-regression-tests-suite/mailinglist-signup-flow.spec.js

```diff
@@ -175,7 +176,7 @@ describe(`Feature - Mailing list sign up : Tests execution date and time : ${new
 		cy.get(".site-footer-top").should("exist").next().should("exist");
 	});
 
-	it("Links through to feedback survey page", function () {
+	it('Links through to "Sign up here"', function () {
 		let rnum = Math.floor(Math.random() * 1000000000 + 1);
```

Comment:
Not sure I follow this test? we start the mailing list sign up and then check we can click on the mailing list sign up bar to restart the sign up?

Reply:
Agree, I've removed it as this test is not required.